src/mkdocstrings/handlers/python.py

```diff
@@ -290,3 +294,30 @@ def rebuild_category_lists(obj: dict) -> None:
     obj["children"] = [child for _, child in obj["children"].items()]
     for child in obj["children"]:
         rebuild_category_lists(child)
+
+
+def sort_object(obj: CollectorItem, sort_style: str) -> None:
```

Comment:
I'll have to point out that it's possible to write unittests for this :D

Reply:
Yes, I wasn't sure how to set that up. Is there a similar test (e.g. for a rendering option) I could use as a template?